src/Symfony/Component/Translation/Formatter/TransMessageFormatter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Formatter;
+
+use Symfony\Component\Translation\MessageSelector;
+
+/**
+ * @author Abdellatif Ait boudad <a.aitboudad@gmail.com>
+ */
+class TransMessageFormatter implements MessageFormatterInterface, PluralMessageFormatterInterface
```

Comment:
`MessageFormatterInterface` is already implicitly included by `PluralMessageFormatterInterface`.